<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>citezinDetailDialog</class>
 <widget class="QDialog" name="citezinDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>515</width>
     <height>652</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 225, 24);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>624</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244,0.94);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Nunito;
font-style: normal;
font-weight: normal;
font-size: 24px;
line-height: 33px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>User Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="firstNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="nationalityLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Nationality</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneNumberLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="nhiLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>NHI</string>
     </property>
    </widget>
    <widget class="QLabel" name="clinicAddressLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Clinic Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="middleNameLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Middle Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="dobLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QLabel" name="genderLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="postCodeLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>270</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Post Code</string>
     </property>
    </widget>
    <widget class="QLabel" name="clinicNameLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Clinic Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="uploadDocLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Upload User Details Document</string>
     </property>
    </widget>
    <widget class="QLabel" name="userDetailsImage">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #000000;
box-sizing: border-box;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 215, 22);
</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>580</y>
       <width>177</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);
</string>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="firstNameEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lastNameEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="middleNameEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nationalityEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneNumberEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nhiEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="clinicAddressEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="genderEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="postCodeEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="clinicNameEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="postCodeLabel_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Ethnicity</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ethnicityEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>691</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(247, 225, 24, 0.4);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>250</width>
      <height>335</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Vaccination Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Clinic Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Clinic Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vaccine Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Batch Number1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Batch Number2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number Of Doses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Vaccine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Second Vaccine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TimeOfBooster</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;

color: #000000;</string>
     </property>
     <property name="text">
      <string>Test Results</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Covid Test Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Covid Test Result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>411</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Nunito;
font-style: normal;
font-weight: normal;
font-size: 24px;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>Covid Test Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;</string>
     </property>
     <property name="text">
      <string>Test Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;</string>
     </property>
     <property name="text">
      <string>Positivie for Coronavirus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;</string>
     </property>
     <property name="text">
      <string>Strain of virus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 16px;</string>
     </property>
     <property name="text">
      <string>upload User Test Document</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>yes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>125</width>
       <height>125</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border: 1px solid #000000;
box-sizing: border-box;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>177</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
